src/format_csharp.js

```diff
@@ -49,14 +53,10 @@ class CSharpFormatter {
       text = `event ${prefix}${toTitleCase(member.alias)}`;
     if (member.kind === 'method' ) {
       args = member.argsArray.slice();
-      const signature = renderSharpSignature(args);
+      if (fullQualified)
+        text += '()';
```

Comment:
Does this mean that `fullQualified` does not include arguments anyway? I think that might be confusing.